migrations/m171028_191358_add_avatar_for_user.php

```diff
@@ -6,7 +6,7 @@ class m171028_191358_add_avatar_for_user extends Migration
 {
     public function safeUp()
     {
-        $this->addColumn('{{%user}}', 'avatar', $this->string(60)->after('fullname'));
+        $this->addColumn('{{%user}}', 'avatar', $this->string(16)->after('fullname'));
```

Comment:
I was talking about increasing it, not decreasing :)

Reply:
How much chars will be enough? default 255?

Reply:
Yep. No need for less chars.

Reply:
Fix this issue.